code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -22,6 +22,9 @@
 	layer = MOB_LAYER
 	max_integrity = 100
 	item_flags = XENOMORPH_HOLDABLE
+	special_desc_requirement = EXAMINE_CHECK_JOB //SKYRAT EDIT
+	special_desc_jobs = list("Scientist, RD") //SKYRAT EDIT
```

Comment:
Make sure to include the entire job name, "Research Director"

Reply:
Global defines protect against this but alright